src/data_gradients/feature_extractors/common/__init__.py

```diff
@@ -1,10 +1,12 @@
 from .image_average_brightness import ImagesAverageBrightness
 from .image_color_distribution import ImageColorDistribution
+from .image_duplicates import ImageDuplicates
 from .image_resolution import ImagesResolution
 from .summary import SummaryStats
 
 
 __all__ = [
+    "ImageDuplicates",
     "ImagesAverageBrightness",
     "ImageColorDistribution",
     "ImagesResolution",
```

Comment:
If you can also import `ImageDuplicates` in the `__init__` of the `feature_extractrs` directory

Reply:
fixed in latest commit.